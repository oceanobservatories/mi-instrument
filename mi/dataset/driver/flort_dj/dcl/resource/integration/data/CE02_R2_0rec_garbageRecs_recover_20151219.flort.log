2015/12/19 00:00:11.625 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2015/12/19 00:00:11.666 18/15	23:57:08	700	591	695	97	460	65	554
2015/12/19 23:47:12.607 19/15	23:44:08	700	549	695	98	460	64	549
2015/12/19 23:47:13.727 19/15	23:44:09	700	564	695	97	460	65	549
2015/12/19 23:47:14.855 19/15	23:44:10	700	596	695	97	460	68	549
2015/12/19 23:47:15.985 19/15	23:44:11	700	564	695	95	460	63	549
2015/12/19 23:47:17.114 19/15	23:44:13	700	563	695	96	460	70	549
2015/12/19 23:47:18.243 19/15	23:44:14	700	578	695	96	460	65	549
2015/12/19 23:47:19.373 19/15	23:44:15	700	571	695	102	460	75	549
2015/12/19 23:47:20.502 19/15	23:44:16	700	559	695	99	460	67	549
2015/12/19 23:47:21.633 19/15	23:44:17	700	572	695	101	460	66	549
2015/12/19 23:47:22.762 19/15	23:44:18	700	555	695	97	460	63	549
2015/12/19 23:47:23.891 19/15	23:44:19	700	597	695	98	460	67	549
2015/12/19 23:47:25.020 19/15	23:44:21	700	565	695	99	460	65	549
2015/12/19 23:47:26.149 19/15	23:44:22	700	567	695	99	460	70	549
2015/12/19 23:48:01.373 [flort:DLOGP2]:Instrument Stopped [Power Off]
