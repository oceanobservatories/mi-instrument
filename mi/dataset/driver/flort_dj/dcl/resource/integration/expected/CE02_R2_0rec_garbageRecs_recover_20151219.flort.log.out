
 subsite | node | sensor | bin | method | time | deployment | id | date_string | dcl_controller_timestamp | driver_timestamp | ingestion_timestamp | internal_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | port_timestamp | preferred_timestamp | provenance | quality_flag | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
---------+------+--------+-----+--------+------+------------+----+-------------+--------------------------+------------------+---------------------+--------------------+-----------------------------+-----------------------------+----------------------------+----------------+---------------------+------------+--------------+-------------------+-----------------+-----------------+----------------+-------------

(0 rows)
